src/containers/SaveTemplateButton/__snapshots__/SaveTemplateButton.test.jsx.snap

```diff
@@ -12,7 +12,9 @@ exports[`<SaveTemplateButton /> renders correctly in disabled state 1`] = `
   <span
     className="d-flex align-items-center justify-content-center"
   >
-    Save Template
```

Comment:
I hope that we can find a way to eliminate these snapshots in favor of better render tests that cover this, but this looks a small change for now is fine

Reply:
Agreed; I was tempted to re-write them all, but I didn't have time.